### PR TITLE
VERSION 1.0.0

### PR DESCRIPTION
**Pull request by stazrad**
Resolves: #44 #153

### PR Safety Checklist:

 - [x] Added the task to the appropriate release doc under **Enhancements** or **Bug Fixes**
 - [x] Bump `package.json` & `package-lock.json` version numbers to appropriate release
 - [x] (optional) All external API changes have been documented
 - [x] (optional) Build docs: `npm run docs`

### CODE REVIEW

File: app/routes/covid/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+#### App.js
+
+The App.js file here is very simple, we're essentialy loading an `ol-kit` map, the same way shown in the start up docs. The only difference is that now we have a `<DataLoader />` component which is handling all the magic.
+
+#### Dataloader.js
+
+Here we are calling ArcGIS using the built-in JavaScript library called `fetch`, and we're getting back a large array of coordinates and confirmed cases corresponding to those coordinates.
+
+We first convert the coordinates from the data to a projection our map can understand using a function from openlayers called [fromLatLon](https://openlayers.org/en/latest/apidoc/module-ol_proj.html)
```
Comment (stazrad) [inline app/routes/covid/README.md:15]: `fromLonLat`

File: app/webpack.config.js [hunk 2]
```diff
@@ -25,7 +25,7 @@ module.exports = {
     alias: {
       // these ensure we don't have duplicate versions on the same page
       '@bayer/ol-kit': path.resolve(__dirname, '../', 'src'),
-      'react-dom': path.resolve(__dirname, '../', 'node_modules/@hot-loader/react-dom')
+      // 'react-dom': path.resolve(__dirname, '../', 'node_modules/@hot-loader/react-dom')
```
Comment (stazrad) [inline app/webpack.config.js:28]: add this back

File: config/jsdoc/template/bundler.js [hunk 3]
```diff
@@ -62,6 +62,7 @@ module.exports = function bundle (Components, out, config) {
     const { displayName, filePath, type } = c.component
     const relativePath = path.relative(absoluteOut, filePath)
     const name = `Component${i}`
+    console.log('relative', name, relativePath)
```
Comment (stazrad) [inline config/jsdoc/template/bundler.js:65]: drop this

File: config/jsdoc/template/bundler.js [hunk 4]
```diff
@@ -79,7 +80,8 @@ module.exports = function bundle (Components, out, config) {
   } catch (error) {
     if(error.output && error.output.length){
       console.log(error.output[1].toString())
+      console.log('ALL ERRORS', error.output)
```
Comment (stazrad) [inline config/jsdoc/template/bundler.js:83]: drop this

File: config/jsdoc/template/bundler.js [hunk 5]
```diff
@@ -79,7 +80,8 @@ module.exports = function bundle (Components, out, config) {
...
     }
-    throw error
+    // throw error
```
Comment (stazrad) [inline config/jsdoc/template/bundler.js:85]: uncomment

Comment (stazrad) [reply config/jsdoc/template/bundler.js:85]: add a console log for the developr to recomment this if build fails

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bayer/ol-kit",
-  "version": "0.18.0",
+  "version": "1.0.0",
```
Comment (stazrad) [inline package.json:3]: 🎉

**Review by glenselle — DISMISSED** [review]
LGTM

**Review by glenselle — APPROVED** [review]
:shipit:

**Pull request merged**